symengine/matrix.h

```diff
@@ -118,90 +118,88 @@ class DenseMatrix : public MatrixBase
     // and throw an exception if a method is not applicable.
 
     // Get and set elements
-    virtual RCP<const Basic> get(unsigned i, unsigned j) const;
-    virtual void set(unsigned i, unsigned j, const RCP<const Basic> &e);
-    virtual vec_basic as_vec_basic() const;
+    RCP<const Basic> get(unsigned i, unsigned j) const override;
+    void set(unsigned i, unsigned j, const RCP<const Basic> &e) override;
+    vec_basic as_vec_basic() const;
```

Comment:
does this change `as_vec_basic()` to be a non-virtual method? Could this possibly break existing user code which relies on overriding this method? (I have to admit that I can't come up with an example where one would want to do so)

Reply:
You are right. This might break user code. We should keep this as virtual since it doesn't exist in the base class. Good catch!